bundles/CreateProject/Form/CreateProjectForm.php

```diff
@@ -46,7 +46,7 @@ public function __construct(JiraService $jiraService, array $options = [])
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-      ->add('project_name', TextType::class, [
+        ->add('project_name', TextType::class, [
```

Comment:
Indentation seems a bit off in this file …

Reply:
Will be handled in coding standards PR.